lib/matplotlib/widgets.py

```diff
@@ -2071,6 +2079,10 @@ def on_select(min: float, max: float) -> Any
         If `True`, the widget can be moved by clicking anywhere within
         its bounds.
 
+    ignore_event_outside : bool, optional
```

Comment:
normal format is `... : bool, default False` (and don't mention the default below)